sdk/ml/azure-ai-ml/azure/ai/ml/operations/_online_deployment_operations.py

```diff
@@ -123,9 +130,20 @@ def begin_create_or_update(
                     msg="VSCode Debug is only support for local endpoints. Please set local to True."
                 )
             if local:
+                if local_enable_gpu:
+                    try:
+                        subprocess.run("nvidia-smi")
```

Comment:
Is nvidia-smi common on Windows?

Reply:
Yes, it is available if you have a Nvidia GPU.